tools/tcpdrop.py

```diff
@@ -194,13 +217,21 @@ def print_ipv6_event(cpu, data, size):
         print("\t%s" % sym)
     print("")
 
+if BPF.tracepoint_exists("skb", "kfree_skb"):
```

Comment:
The `reason` field may not exist, this would cause compilation error.

Reply:
I try add `BPF.kernel_enum_has_field()` API